### DISCUSSION TITLE
研究生word模板双语标题问题

### DISCUSSION THREAD

**Discussion by tengxiao24** [sikouhjw/gdutthesis-Word#6] [Q&A] [answered]
作者您好，关于双语标题的设置，我阅读Wiki.md文件  选择图文集中的一、二、三级标题 不知道对应的英语哪里写，望赐教。

**Comment by sikouhjw** [comment, marked as answer]
Wiki 的《TC 域代码》[^1]部分

[^1]: https://github.com/sikouhjw/gdutthesis-Word/wiki#tc-%E5%9F%9F%E4%BB%A3%E7%A0%81

**Comment by sikouhjw** [comment]
如果解决了可以 Close issue。
